reframe/core/schedulers/slurm.py

```diff
@@ -127,6 +127,8 @@ def completion_time(self, job):
                 log=False
             )
 
+        # Use `list(re.finditer())`` instead of `re.findall()` in order
+        # to use the named groups. `re.findall` returns a list of tuples.
```

Comment:
Keep the comment simple so as to fit in a single line. Also don't use formatting characters for such code comments.

```suggestion
        # We need the match objects, so we have to use finditer()
```